sdk/hybridkubernetes/azure-mgmt-hybridkubernetes/README.md

```diff
@@ -4,13 +4,6 @@ This is the Microsoft Azure HybridKubernetes Management Client Library.
 This package has been tested with Python 2.7, 3.5, 3.6, 3.7 and 3.8.
 For a more complete view of Azure libraries, see the [Github repo](https://github.com/Azure/azure-sdk-for-python/)
```

Comment:
question: some of these point to https://github.com/Azure/azure-sdk-for-python/tree/master/sdk
and some to the root https://github.com/Azure/azure-sdk-for-python - is there any specific pattern? - don't see much value in changing these for consistency, so don't bother changing them all

Reply:
I am not sure, these were already there so i updated only the ones which are broken. i am not sure what the standard should be